scripts/run_docker_build.sh

```diff
@@ -52,7 +52,7 @@ export LANG=en_US.UTF-8
 # These were specific to installing matplotlib. I really want to avoid doing this if possible, but in some cases it
 # is inevitable (without re-implementing a full OS), so I also really want to ensure we can annotate our recipes to
 # state the build dependencies at OS level, too.
-yum install -y libXext libXrender libSM tk libX11-devel
+yum install -y libXext libXrender libSM tk libX11-devel mesa-libGL-devel
```

Comment:
Should we move all the X11 stuff into the `Dockerfile`? There was some discussion about adding `yum` dependencies to recipes at one point. That may actually be better in some ways. The main reason being if one needs to build a Docker image, they will know what needs to be `yum` installed or not.